appserver/admingui/jmx-monitoring-plugin/src/main/resources/jmxmonitoring/jmxmonitoring.jsf

```diff
@@ -2,7 +2,7 @@
 
 DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 
-   Copyright (c) [2017-2018] Payara Foundation and/or its affiliates. All rights reserved.
+   Copyright (c) [2017-2019] Payara Foundation and/or its affiliates. All rights reserved.
```

Comment:
I get a NPE on visiting this page:

Caused by: java.lang.NullPointerException
	at com.sun.jsftemplating.handlers.UtilHandlers.mapPut(UtilHandlers.java:332)

Reply:
As do I